Nosenti/Innocent-Ingabire-portfolio
# 'ft sigin page and responsiveness h4jj htkf'

#### What does this PR do?
Create the responsive sign-in page
Add responsiveness to Contact and Landing pages
Add Figma file

#### Description of Task to be completed
Created the responsive sign-in page
Addedresponsiveness to Contact and Landing pages
AddedFigma file of the sign-in page

#### How should this be manually tested?
You can clone the folder and run HTML files. Inspect using any browser (Chrome is highly recommended) and adjust the window to different screen sizes to check the responsiveness of different pages. 

#### What are the relevant stories?
<blockquote class="trello-card"><a href="https://trello.com/c/H4jjHTKF/3-user-signin-page">User Signin Page</a></blockquote><script src="https://p.trellocdn.com/embed.min.js"></script>

#### Screenshots 
![Signin Page](https://user-images.githubusercontent.com/36986840/88337369-edd04980-cd36-11ea-80be-47e2e3d9efdb.jpg)
